ndcube/ndcube_sequence.py

```diff
@@ -239,6 +241,46 @@ def explode_along_axis(self, axis):
         # creating a new sequence with the result_cubes keeping the meta and common axis as axis
         return self._new_instance(result_cubes, common_axis=new_common_axis, meta=self.meta)
 
+    def combine_cubes(self, axis=0, common_wcs_index=0):
+        """
+        Reprojects all `~ndcube.NDCube` objects to a common WCS, and stacks the data together
+        to return a single (N+1)-dimensional `~ndcube.NDCube` with an associated Compound WCS
+        containing extra axes corresponding to the sequence axis.
+
+        Parameters
+        ----------
+        axis: `int`, optional
+            The axis along which the data is to be stacked.
```

Comment:
This is a little confusing. What this is is the *array* axis we are stacking along.

I do wonder if we even want this to be a configurable parameter. If we are being purists about it then the slowest varying coordinate should go last (which it is a safe assumption is the sequence dimension). I would propose maybe just dropping this parameter all together. I wonder if @DanRyanIrish or @wtbarnes have differing opinions on that though.

Reply:
For now, I've dropped the axis parameter to make things a little simpler while computing the pixel to world axis mapping.